### ISSUE TITLE
Antennny MK7 Documentation

### ISSUE THREAD

**Issue by aeonik** [RedBalloonShenanigans/antenny#100]
Hello,
I am trying to setup the Antenny MK7 from the Space access kit, but the getting started guide seems to be incomplete. I am not sure which headers need to be soldered, or where the connections need to be made. The gimbal that came with the kit also seems to be sized for a different board.

Regards,
Aeonik

**Comment by SometimesCake** [comment]
Yeah the gimbal will need some mount point created to interface with the board. The jumpers you are looking for are on the main page, just scroll down and see the first picture. https://github.com/RedBalloonShenanigans/antenny Hopefully someone will create an STL. Looks like both boards need mounting brackets.

Basically all 5 jumpers on the back in the outline need to be connected.

**Comment by aeonik** [comment]
I got the jumpers on fine. It's the next steps that I am unsure of.

For example, what to do with the IMU, or which headers need to be soldered on in order to drive the gimble.

I am also curious if there are recommended antennas.

**Comment by dannyp303** [comment]
Hello, next steps should be to assemble your "platform". This entails plugging the servo gimbal into the PWM controller pins and the IMU to the I2C pins. The pins used for this are configurable and are described in [the antenny config](https://github.com/RedBalloonShenanigans/antenny/blob/master/nyansat/station/configs/antenny/default.json), or can be set during runtime using the command `api.antenny_config.set(<config member>, <pythonic_value>)`. Hope this helps!

**Comment by cjearls95** [comment]
Is there any amount of breakout headers that should be populated for the motors? I planned on populating at minimum 2 sets of the groups of 4 along with the two ESP breakout header lengths and the two i2c breakouts.

Edit: Looking over the schematic It looks like it shouldn't matter which/how many are populated with headers. Is there any way RedBalloon suggests populating them?

**Comment by dannyp303** [comment]
You are correct about which/how many. I currently have mine setup with PWM indexes 0 and 1 soldered.  These are the 3 pin connectors on the side of the board labeled PWM, +5V and GND. On your servo gimbal, the brown wire is GND. You can populate as many as you want, but bare minimum would be 2 x 6 pins for the servos and 5 pins total for VCC, GND, SCL (ESP pin 26), SDA (ESP pin 27) and RST(ESP pin 4) for the IMU, those pinouts are printed on the back of the board.

**Comment by cjearls-soter** [comment]
Looking over the newly updated documentation, it looks like the picture of the ESP R14 is soldered for the WiFi, but the document says R15.

**Comment by cjearls95** [comment]
Looking over the newly updated documentation, it looks like the picture of the ESP R14 is soldered for the WiFi, but the document says R15.

**Comment by dannyp303** [comment]
There are two resistors, R14 and R15. R14 will connect to the antenna connector seen next to it, for the stock configuration a compatible antenna is required. R15 connects to the PCB antenna, but the resistor must be swapped to use it

**Comment by cjearls95** [comment]
Awesome thank you. Additionally, what size screws did you guys use to secure the IMU and the antenna base? Don't want to damage my gimble. Would it be possible to add this into the GitHub documentation?

**Comment by dannyp303** [comment]
I'll be putting up an assembly guide today, however the way you fix the board/imu to the gimbal is mostly up to you. I just used some M3 standoffs, washers and a bit of hot glue.
